src/lib/HasuraClient.ts

```diff
@@ -0,0 +1,25 @@
+import { ApolloClient, createHttpLink, InMemoryCache } from "@apollo/client";
+import { setContext } from "@apollo/client/link/context";
+
+const HASURA_TOKEN_KEY = "hasura_token";
+
+const httpLink = createHttpLink({
+  uri: process.env.REACT_APP_HASURA_URL,
+});
+
+const authLink = setContext((_, { headers }) => {
+  const token = localStorage.getItem(HASURA_TOKEN_KEY);
```

Comment:
Don't think we don't need to store the hasura token in the local storage. Since it expires so quickly and needs to be refreshed regularly we can just keep it in memory